content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+
+.. important::
+   When the :guilabel:`Booth Management` setting is activated, a new *Product Type* becomes
+   available on all product forms: *Event Booth*.
+
+   This is important because every created booth **must** be assigned a *Booth Category* on their
+   respective booth form, and every booth category **must** have an *Event Booth* product assigned
+   to it.
+
+Booth categories
+================
+
+With the *Booth Management* setting activated in the *Events* app, a *Booth Categories* option
+becomes available in the :guilabel:`Configuration` menu.
+
+To access the :guilabel:`Booth Category` dashboard, go to :menuselection:`Events app -->
+Configuration --> Booth Categories`. Doing so reveals a list of all the booth categories that have
+been created in the database.
+
+.. image:: event_booths/booth-category-page.png
+   :align: center
+   :alt: The Booth Category page in the Odoo Events application.
+
+On the :guilabel:`Booth Category` page, the following information for each booth category is listed:
+
+- :guilabel:`Name`: the name of the booth category.
+- :guilabel:`Create Sponsor`: if the checkbox in this column is ticked, when a booth with this
+  category is booked, a sponsor is created for the user who booked it.
+- :guilabel:`Product`: the *Event Booth* product associated with that specific booth category.
+- :guilabel:`Price`: the price of a booth in that booth category.
+
+When the :guilabel:`additional options` :icon:`oi-settings-adjust` icon, located to the far-right of
+the column titles, is clicked, a drop-down menu of additional column options appears. From the
+resulting drop-down menu, tick the checkbox beside :guilabel:`Sponsor Level` and/or
+:guilabel:`Sponsor Type` to reveal those columns on the :guilabel:`Booth Category` page.
+
+To edit an existing booth category, select it from the list, and proceed to make any desired
+modifications from the event category form.
+
+Create booth category
+---------------------
+
+To create a booth category from the :guilabel:`Booth Category` page, click the :guilabel:`New`
+button in the upper-left corner to reveal a blank booth category form.
+
+.. image:: event_booths/booth-category-form.png
+   :align: center
+   :alt: A typical booth category form in the Odoo Events application.
+
+Start by entering a name for the booth category in the top :guilabel:`Booth Category` field. This is
+a **requried** field.
+
+To add a corresponding image to the booth category (e.g. a sample photo of how the booth looks),
+click the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers over the
+camera placeholder in the upper-right corner of the booth category form. When clicked, proceed to
+upload the desired image to the booth category form, if needed.
+
+In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product` to the
+category, and it **must** have *Event Booth* set as the *Product Type* on the product form.
```

Comment:
optional, for brevity
```suggestion
In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product`, with *Event Booth* set as the *Product Type* on the product form.
```

Reply:
I know that my version is slightly longer, but I think it's more just a *tiny bit* easier for the reader to comprehend. So, if it's okay with you, I think I'm going to stick with the current wording. However, if you feel like I should rework this portion, just let me know, and I'll make the necessary adjustments.